Orange/widgets/classify/owclassificationtree.py

```diff
@@ -65,6 +65,26 @@ def get_learner_parameters(self):
             ("maximum depth {}".format(self.max_depth), self.limit_depth)) if c) or "None"
         return items
 
+    def init_code_gen(self):
+        def run():
+            ow.set_data(input_data)
```

Comment:
I've started adding `__repr__` to Orange native objects in a new branch, https://github.com/Ameobea/orange3/commits/code_gen_repr and will expand those efforts to learners, classifiers, and other internals as I progress.  

I'll close this PR and (maybe) make another WIP PR with the updated code once I'm finished.